### ISSUE TITLE
Request for new skey_file option in .conf files

### ISSUE THREAD

**Issue by aanderse** [duosecurity/duo_unix#172]
## Summary
I would like if `login_duo.conf` and `pam_duo.conf` would accept a new `skey_file` field. The intention of the field is that `login_duo.conf` and `pam_duo.conf` wouldn't need to store the actual value of `skey` anymore, but could reference the secret key via the `skey_file` field. The benefit to this is that the `.conf` files wouldn't need to be private anymore. Often times it can be desirable to have configuration available to general users on a system, just keeping credentials private in a separate file that has the appropriate permissions.

**Comment by boyangz** [comment]
Hey @aanderse!

Thanks for your suggestion. We discussed this issue internally, and at this time feel that this is a bit of a high risk / low reward type of scenario from a security perspective. It would require admins to manage file permissions for two separate config files, and we can't think of many scenarios where you would want a user on the machine to be able to access the general config, but not the skey. 

However, I don't think this feature would be super difficult to implement; I would encourage you to fork our repo and try it out yourself! I think most of the changes would be isolated to the parts of the code base that load the Duo config. 

In addition, I would encourage you to submit a feature request with our product team. These enable our product team to track popular customer requests over time, and we could revisit this request if we started hearing it from a large population of our Duo Unix users. 

Thanks you for contributing to Duo's open source libraries!
